configure-places-in-the-sdk/places-extension/using-places-without-active-region-monitoring.md

```diff
@@ -0,0 +1,415 @@
+
+{% tabs %}
+
+{% tab title="Android (Java)" %}
+
+Example implementation using a [`BroadcastReceiver`](https://codelabs.developers.google.com/codelabs/background-location-updates-android-o/index.html?index=..%2F..index#5) in Android:
+
+```java
+public class LocationBroadcastReceiver extends BroadcastReceiver {
+    static final String ACTION_LOCATION_UPDATE = "locationUpdate";
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        if (intent == null || context == null) {
+            return;
+        }
+
+        final String action = intent.getAction();
+        if (!ACTION_LOCATION_UPDATE.equals(action)) {
+            return;
+        }
+
+        LocationResult result = LocationResult.extractResult(intent);
+        if (result == null) {
+            return;
+        }
+
+        Location currentLocation = result.getLastLocation();
+        if (currentLocation == null) {
+            return;
+        }
+
+        // ask the Places SDK for the 10 nearest Points of Interest based on the user's location
+        Places.getNearbyPointsOfInterest(currentLocation, 10,
+                new AdobeCallback<List<PlacesPOI>>() {
+                    @Override
+                    public void call(List<PlacesPOI> pois) {
+                        // pois is the 10 nearest POIs based on the location
+                    }
+                }, new AdobeCallback<PlacesRequestError>() {
+                    @Override
+                    public void call(PlacesRequestError placesRequestError) {
+                        // Look for the placesRequestError and handle the error accordingly
+                    }
+                });
+    }
+}
+```
+{% endtab %}
+
+{% tab title="iOS (Objective-C)" %}
+
+Example implementation from a [`CLLocationManagerDelegate`](https://developer.apple.com/documentation/corelocation/cllocationmanager?language=objc) method [`locationManager:didUpdateLocations:`](https://developer.apple.com/documentation/corelocation/cllocationmanagerdelegate/1423615-locationmanager?language=objc):
+
+```objectivec
+- (void) locationManager:(CLLocationManager*)manager didUpdateLocations:(NSArray<CLLocation*>*)locations {
+    // ask the Places SDK for the 10 nearest Points of Interest based on the user's location
+    [ACPPlaces getNearbyPointsOfInterest:[locations lastObject] limit:10 callback:^(NSArray<ACPPlacesPoi *> * _Nullable nearbyPoi) {
+        // nearbyPoi is the 10 nearest POIs based on the location
+    } errorCallback:^(ACPPlacesRequestError result) {
+        // log the error if we got one
+        NSLog(@"error: %lu", (unsigned long)result);
+    }];
+}
+```
+{% endtab %}
+
+{% tab title="iOS (Swift)" %}
+
+Example implementation from a [`CLLocationManagerDelegate`](https://developer.apple.com/documentation/corelocation/cllocationmanager) method [`locationManager(_:didUpdateLocations:)`](https://developer.apple.com/documentation/corelocation/cllocationmanagerdelegate/1423615-locationmanager):
```

Comment:
Please reword to:
"Here is a implementation from a [`CLLocationManagerDelegate`](https://developer.apple.com/documentation/corelocation/cllocationmanager) method [`locationManager(_:didUpdateLocations:)`](https://developer.apple.com/documentation/corelocation/cllocationmanagerdelegate/1423615-locationmanager):"